### PR TITLE
Settings to ParSets

### PR DESCRIPTION
**Pull request by kbwestfall**
These are the edits X and I made to work with [PYPIT #400](https://github.com/PYPIT/PYPIT/pull/400).

### CODE REVIEW

**Comment by jhennawi** [comment]
Looks great guys. Thanks for the hard work on this.

File: pypit_files/keck_lris_blue_long_400_3400_d560.pypit [hunk 1]
```diff
@@ -1,31 +1,30 @@
...
+    [[biasframe]]
+        useframe = overscan
+        [[[process]]]
+            sigrej = 20.
+    [[pixelflatframe]]
+        number = 1
+    [[traceframe]]
+        number = 3
+    [[flatfield]]
+        frame = /home/xavier/Projects/Python/PYPIT-development-suite/CALIBS/PYPIT_LRISb_pixflat_B400_2x2_15apr2015.fits.gz
```
Comment (rcooke-ast) [inline pypit_files/keck_lris_blue_long_400_3400_d560.pypit]: Do you think it is possible to make this path more generalised?

File: pypit_files/keck_lris_blue_long_600_4000_d560.pypit [hunk 2]
```diff
@@ -1,33 +1,26 @@
...
-output sorted lris_blue_long_600_4000_d560
+[calibrations]
+    [[biasframe]]
+        useframe = overscan
+    [[pixelflatframe]]
+        number = 0
+    [[traceframe]]
+        number = 1
+    [[flatfield]]
+        frame = /home/xavier/Projects/Python/PYPIT-development-suite/CALIBS/PYPIT_LRISb_pixflat_B600_2x2_17sep2009.fits.gz
```
Comment (rcooke-ast) [inline pypit_files/keck_lris_blue_long_600_4000_d560.pypit]: likewise

File: pypit_files/shane_kast_red_600_7500_d55.pypit [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
+        number = 3
+    [[standardframe]]
+        useframe = r136.fits.gz
+    [[tilts]]
+        method = spca
+        params = 1, 1, 1
 
 # Read in the data
 data read
  ~/Dropbox/PYPIT/TEST_SUITES/Kast_red/600_7500_d55/r1*.fits.gz
```
Comment (rcooke-ast) [inline pypit_files/shane_kast_red_600_7500_d55.pypit]: Do we still distribute via dropbox?

File: pypit_files/wht_isis_blue_long_r300b_d5300.pypit [hunk 4]
```diff
@@ -1,34 +1,41 @@
...
-science extraction maxnumber 3
+[calibrations]
+    [[biasframe]]
+        number = 2
+    [[pixelflatframe]]
+        number = 3
+    [[traceframe]]
+        number = 3
+    [[wavelengths]]
+        IDpixels  = 871.062,901.7719,1930.0048,2451.620,3364.25658,3886.125
```
Comment (rcooke-ast) [inline pypit_files/wht_isis_blue_long_r300b_d5300.pypit]: Is this still needed, even with the `general` algorithm in arclines?

File: pypit_files/keck_lris_blue_long_400_3400_d560.pypit (hunk 1 above)
Comment (kbwestfall) [reply pypit_files/keck_lris_blue_long_400_3400_d560.pypit]: Yeah, I think so, but I haven't worked out the details yet.  This is raised as [PYPIT #402](https://github.com/PYPIT/PYPIT/issues/402).  I haven't set a milestone for this, but we could make it 1.0.

File: pypit_files/shane_kast_red_600_7500_d55.pypit (hunk 3 above)
Comment (kbwestfall) [reply pypit_files/shane_kast_red_600_7500_d55.pypit]: These paths get overwritten by `pypit_test`.  This is a question for @profxj, I think.

File: pypit_files/wht_isis_blue_long_r300b_d5300.pypit (hunk 4 above)
Comment (kbwestfall) [reply pypit_files/wht_isis_blue_long_r300b_d5300.pypit]: Good question.  Another one for @profxj.

File: pypit_files/shane_kast_red_600_7500_d55.pypit (hunk 3 above)
Comment (profxj) [reply pypit_files/shane_kast_red_600_7500_d55.pypit]: They are over-written

File: pypit_files/wht_isis_blue_long_r300b_d5300.pypit (hunk 4 above)
Comment (profxj) [reply pypit_files/wht_isis_blue_long_r300b_d5300.pypit]: I fear it is as I have yet to do CuNe properly.

**Comment by kbwestfall** [comment]
Sounds good.  I'll wait for @rcooke-ast to approve [PYPIT #400](https://github.com/PYPIT/PYPIT/issues/400) and then merge both at the same time.

**Comment by kbwestfall** [comment]
PYPIT PR #400 is now merged.  Syncing this...

**Pull request merged**